.github/workflows/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
+
+on: pull_request
+
```

Comment:
This linter requires that you have `2 spaces` for you .css files with a `new line`

e.g 
```
.header {
  color: #000000;
}
```

Reply:
Made changes to spacing